client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -184,7 +180,7 @@ private static void doTestSourceExists(BiFunction<String, String, GetRequest> re
         if (randomBoolean()) {
             boolean realtime = randomBoolean();
             getRequest.realtime(realtime);
-            if (realtime == false) {
+            if (!realtime) {
```

Comment:
In the es code base we prefer use `== false` instead of `!`. The reason is that this is more visible when reading the code. A `!` can easily be missed or forgotten to be added.

Reply:
thanks for the clarification! fixed it.